htdocs/public/payment/newpayment.php

```diff
@@ -473,7 +474,7 @@
 					'customer' => $customer->id,
 					'source' => $card,
 					'statement_descriptor' => dol_trunc(dol_trunc(dol_string_unaccent($mysoc->name), 6, 'right', 'UTF-8', 1).' '.$FULLTAG, 22, 'right', 'UTF-8', 1)     // 22 chars that appears on bank receipt
-				));
+				),array("idempotency_key" => "$ref","stripe_account" => "$stripeacc"));
```

Comment:
Note that $ref may not be defined. What happen for such payment if we add an idempotency_key flag with a '' value ?
Is Stripe ok with this ?